src/lib.rs

```diff
@@ -636,6 +649,31 @@ storage_types! {
     }
 }
 
+storage_types! {
+    types: Complex;
+
+    impl crate::Conversion<V> for V {
+        type T = V;
+
+        #[inline(always)]
+        fn conversion(&self) -> Self::T {
+            *self
+        }
+    }
+
+    impl crate::ConversionFactor<V> for V {
+        #[inline(always)]
+        fn powi(self, e: i32) -> Self {
+            self.powi(e)
+        }
+
+        #[inline(always)]
+        fn value(self) -> V {
+            self
+        }
+    }
+}
```

Comment:
Use `Complex<T>`'s underlying type when implementing `Conversion<V>` and `ConversionFactor<V>`. These changes essentially say that the conversion factor type used to convert between units for `Complex<T>` is `T`. See the other suggestion where the definition for `VV` is added.

EDIT: The diff doesn't come out very good. The main changes are `type T = VV;` and `impl crate::ConversionFactor<V> for VV {`

```suggestion
storage_types! {
    types: Complex;

    impl crate::Conversion<V> for V {
        type T = VV;

        #[inline(always)]
        fn constant(op: crate::ConstantOp) -> Self::T {
            match op {
                crate::ConstantOp::Add => -<Self::T as crate::num::Zero>::zero(),
                crate::ConstantOp::Sub => <Self::T as crate::num::Zero>::zero(),
            }
        }

        #[inline(always)]
        fn conversion(&self) -> Self::T {
            *self
        }
    }

    impl crate::ConversionFactor<V> for VV {
        #[inline(always)]
        fn powi(self, e: i32) -> Self {
            self.powi(e)
        }

        #[inline(always)]
        fn value(self) -> V {
            self
        }
    }
}
```